### PR TITLE
feat: darken s:palette.comment and s:palette.fg to make it more friendly to eyes

### PR DESCRIPTION
**Pull request by shiyanhui**
Hi,

Recently I'm looking for an alternative to solarized, and I found elly.vim. It's comfortable and I like it very much.  But one problem is that the foreground color is too bright which will hurt eyes heavily. So I change it a bit in this repo and elly-iterm2.

### CODE REVIEW

**Review by ulwlu — CHANGES_REQUESTED** [review]
Hi @shiyanhui , Thank you so much for your PR!
I think this PR makes sense very much because a similar issue has been posted https://github.com/ulwlu/elly.vim/issues/13 a few day agos by @d86leader.

At that time I decided not to change, but now, I realized many people feel same way. 
I should merge this PR.

Before merging, would you check my change-suggestions below?
I'd like to adjust the color palette along with **less red, more green** and **complementary each other**.

Theme will be like this.
ss 4

File: colors/elly.vim [hunk 1]
```diff
@@ -13,7 +13,7 @@ let g:colors_name = "elly"
 " ----------------------------------------------------------------------------
 let s:palette = {}
 let s:palette.bg        = "#111a1f"
-let s:palette.comment   = "#797E80"
+let s:palette.comment   = "#5A5A5A"
```
Comment (ulwlu) [inline colors/elly.vim:16]:
```suggestion
let s:palette.comment   = "#545759"
```

File: colors/elly.vim [hunk 2]
```diff
@@ -29,7 +29,7 @@ let s:palette.panel     = "#14191F"
 let s:palette.guide     = "#2D3640"
 let s:palette.line      = "#151A1E"
 let s:palette.selection = "#253340"
-let s:palette.fg        = "#c0c7ca"
+let s:palette.fg        = "#9A9A9A"
```
Comment (ulwlu) [inline colors/elly.vim:32]:
```suggestion
let s:palette.fg        = "#999FA1"
```

File: colors/elly.vim (hunk 1 above)
Comment (shiyanhui) [reply colors/elly.vim:16]: I think it's ok to change comment to #545759.

File: colors/elly.vim (hunk 2 above)
Comment (shiyanhui) [reply colors/elly.vim:32]: It's still a little bright but can be acceptable.

**Review by ulwlu — APPROVED** [review]
Thank you very much! This is really good turning of this theme.
Merging.

**Pull request merged**